.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,37 @@
 <!--- Please review your changes in preview mode -->
 <!--- Provide a general summary of your changes in the Title above -->
 
-#### Issue-ID
+#### Issue-ID (If there is an associated issue)
 <!--- Replace XXXX below (2 places) with the actual JIRA id -->
 * **[PP-XXXX](https://pbspro.atlassian.net/browse/PP-XXXX)**
 
-#### Problem description
-* Describe the issue being fixed
+#### Problem description 
+* Clearly describe the problem as it affects the customer
+* Clearly describe the steps to reproduce the problem
+* If the issue is intermittent and not easily reproducible, describe the situation/scenarios/configurations where it was likely triggered
+* Specify the exact platform where the bug is observed, for example CentOS 7.2
+
+Make it easy to understand and reproduce the problem that the customer sees.  You can include short illustrative excerpts from log files or terminal sessions.  Do not include large sections of log files or terminal sessions.  The description should never be empty.
+
+#### Affected Platform(s) and PBS Version
+* Specify the exact platform where the bug is observed or the feature is supported, for example CentOS 7.2
+* For bugs, specify the version where the the bug is observed
 
 #### Cause / Analysis
 * Briefly describe root cause/analysis of the problem
 
 #### Solution description
 * Please describe your changes in detail for reviewers
 
+#### External design (if one exists)
+* Add a link to your external design **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**
+
+#### Link to Forum discussion (if it exists)
+* Add a link to the discussion on the Forum **[pbspro community forum](http://community.pbspro.org/)**
+
+#### Test results, including logs
+* Add your test logs and other test results.  If you did manual tests, explain why PTL is not appropriate.
```

Comment:
I think you should change the wording from "Add" to "Attach" and say "test log files" because we don't want people to copy paste the logs themselves here.

Reply:
Done